js/scalecreature/scalecreature-scaler-cr/scalecreature-scaler-cr-dpr.js

```diff
@@ -104,7 +104,7 @@ export class CrScalerDpr extends CrScalerBase {
 				return `${prefix}${adjDpr}${suffix}`;
 			});
 
-			// track attribute adjustment requirements (unused except for dbgging)
+			// track attribute adjustment requirements (unused except for debugging)
```

Comment:
these `dbg` ones are all intentional; please revert/maintain

Reply:
Think everything under 4 characters is ignored so `dbg` throughout is unaffected.
This one was just a single instance shortened long form. Will revert.

Reply:
![image](https://github.com/user-attachments/assets/2deaca9b-5f68-43f5-8f0d-99127fdf1f38)
Lol it's like "no pls put it back"